plugins/broker-redis/src/static/redis.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Inject, Injectable } from "@nestjs/common";
+import { ClientRedis } from "@nestjs/microservices";
+import { REDIS_BROKER_CLIENT } from "./constants";
+
+@Injectable()
+export class RedisService {
```

Comment:
Let's rename this as RedisProducerService and add an example in the README to show how to use it to create a message.